### PR TITLE
Fix format description for TYPE_CONTEXT

### PR DESCRIPTION
**Pull request by rodrmoya**

### CODE REVIEW

File: mono/profiler/proflog.c [hunk 1]
```diff
@@ -232,19 +232,20 @@ typedef struct _LogBuffer LogBuffer;
  * [pointer: sleb128] pointer of the metadata type depending on mtype
  * if mtype == TYPE_CLASS
  *	[image: sleb128] MonoImage* as a pointer difference from ptr_base
- *  [flags: uleb128] must be 0
+ *	[flags: uleb128] must be 0
  * 	[name: string] full class name
  * if mtype == TYPE_IMAGE
- *  [flags: uleb128] must be 0
+ *	[flags: uleb128] must be 0
```
Comment (alexrp) [inline mono/profiler/proflog.c]: indentation doesn't line up

**Comment by alexrp** [comment]
Otherwise LGTM.

File: mono/profiler/proflog.c (hunk 1 above)
Comment (rodrmoya) [inline mono/profiler/proflog.c]: The change is supposed to fix indentation, as the lines below are using tabs, not spaces, as the lines being changed do

Comment (alexrp) [inline mono/profiler/proflog.c]: sure, but you are missing a space before the tab

Comment (rodrmoya) [inline mono/profiler/proflog.c]: oh ok, fixed

**Pull request merged**